relbench/data/dataset.py

```diff
@@ -21,13 +21,17 @@ def __init__(
         db: Database,
         val_timestamp: pd.Timestamp,
         test_timestamp: pd.Timestamp,
+        max_eval_time_frames: int,
```

Comment:
Can we add a doc-string on `Dataset` to document all these variables? Example: https://github.com/pyg-team/pytorch-frame/blob/master/torch_frame/gbdt/gbdt.py#L18-L38